### ISSUE TITLE
Model Training Parameter Casting

### ISSUE THREAD

**Issue by ERIK-KE** [PraiseTheDarkFlo/CellSePi#69]
Upon changing the parameters like learning rate in the training configuration, training is not executed properly and the following message is displayed:

`Something went wrong while training: ufunc 'multiply' did not contain a loop with signature matching types (dtype('<U4'), dtype('float64')) -> None`

This is caused by missing casting of parameters after changing by user. In particular, the value of `self.learning_rate` after the user change is for example `'0.01'` (string) instead of `0.01`(float). This is also the case for other parameters.

Thanks for solving this!

**Comment by SirHenry10** [comment]
Fixed issue in commit [63e97d7](https://github.com/PraiseTheDarkFlo/CellSePi/commit/63e97d71ac1e92a96a8b863894cfad45de6fbb83)
